.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I have no idea why this file was generated.  Any idea??

Reply:
You probably hit F5 on this workspace, even though you can't really 'F5' an npm package